wp_api/src/request.rs

```diff
@@ -278,6 +288,11 @@ impl WpNetworkHeaderMap {
             .and_then(|h| h.parse().ok())
     }
 
+    pub fn insert(&mut self, header_name: HeaderName, header_value: String) {
```

Comment:
Added because `from_map` doesn't properly handle cases where the header value (validly) contains a comma